packages/modules/viewer-react/src/services/iModel/iModelViewerHelper.ts

```diff
@@ -0,0 +1,91 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import type {
+  BlankConnectionProps,
+  IModelConnection,
+} from "@itwin/core-frontend";
+import { BlankConnection } from "@itwin/core-frontend";
+
+import type {
+  BlankConnectionInitializationProps,
+  ModelLoaderProps,
+  RequiredViewerConnectionProps,
+} from "../../types";
+import { ConnectionType } from "../../types";
+import { openLocalIModel, openRemoteIModel } from "./IModelService";
+
+/**
+ * Create a blank connection with default props
+ * @param BlankConnectionInitializationProps
+ * @returns BlankConnection
+ */
+export const createBlankConnection = ({
+  iTwinId,
+  blankConnectionProps,
+}: BlankConnectionInitializationProps) =>
+  BlankConnection.create({
+    iTwinId,
+    ...blankConnectionProps,
+  });
+
+export const getConnectionType = ({
```

Comment:
i was on a TS craze when i did my refactor for 3.0; wondering if we should use type guards instead

Reply:
Yep, I can clean this up.